model/src/w3initmd.F90

```diff
@@ -459,7 +462,7 @@ SUBROUTINE W3INIT ( IMOD, IsMulti, FEXT, MDS, MTRACE, ODAT, FLGRD,  FLGR2, FLGD,
     !/ ------------------------------------------------------------------- /
     !/ Parameter list
     !/
-    INTEGER, INTENT(IN)           :: IMOD, MDS(13), MTRACE(2),      &
+    INTEGER, INTENT(IN)           :: IMOD, MDS(15), MTRACE(2),      &
```

Comment:
There are a couple of changes needed in the mesh cap code to align w/ the increased array size of MDS. See https://github.com/NOAA-EMC/WW3/blob/29063ec7b631d3a967172e8c45c5af13a18e0a82/model/src/wav_comp_nuopc.F90#L485

https://github.com/NOAA-EMC/WW3/blob/29063ec7b631d3a967172e8c45c5af13a18e0a82/model/src/wav_shel_inp.F90#L53

Reply:
Great catch! Thank you @DeniseWorthen